torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -120,10 +120,8 @@ inline void rebase_history(
   if (grad_fn) {
     for (auto& var : vars) {
       if (var.defined()) {
-        // TODO: eliminate const_cast
-        // NOLINTNEXTLINE(bugprone-use-after-move)
         auto output_nr = grad_fn->add_input_metadata(var);
-        impl::rebase_history(var, {std::move(grad_fn), output_nr});
+        impl::rebase_history(var, {grad_fn, output_nr});
```

Comment:
Why remove this? Seems fine? The other use of grad_fn is in an else statement?

Reply:
grad_fn is moved in a loop and may be used after the next iteration. It is safer not to move it.